src/Netpbm.jl

```diff
@@ -84,6 +139,33 @@ end
     dat
 end
 
+@noinline function readtextio!(io, dat::AbstractMatrix{T}) where {T}
+    h, w = size(dat)
+    for i = 1:h, j = 1:w  # io is stored in row-major format
```

Comment:
```suggestion
    axh, axw = axes(dat)
    for i in axh, j in axw  # io is stored in row-major format
```
Julia is more general now than when this package was first written. (It can handle arrays whose indexing doesn't start at 1, no reason not to support that.)

This change is optional; I won't comment on it further, and I'll let you decide whether you want to make it and, if so,whether to do it elsewhere too. (Can also be done at a later time.)

Reply:
That is an excellent suggestion. I really must get used to using the generalisations that are now available.

It doesn't seem very useful to change only in one place, so I will commit the current PR, and follow up later with changes to indexing wherever it makes sense. That will make sure I can test it properly beforehand.